landoui/template_helpers.py

```diff
@@ -178,7 +178,10 @@ def linkify_revision_urls(text: str) -> str:
 def linkify_revision_ids(text: str) -> str:
     """Linkify `D1234` to a proper Phabricator URL."""
```

Comment:
And provide examples instead.

```suggestion
    """Linkify references to revisions to proper Phabricator URLs."""
```